KoAR.SaveEditor/Views/MainWindow.xaml

```diff
@@ -289,17 +289,7 @@
                         <TabControl.ItemTemplate>
                             <DataTemplate DataType="{x:Type constructs:DataContainer}">
                                 <StackPanel Orientation="Horizontal">
-                                    <views:EquipmentCategoryIcon Category="{Binding Data}" VerticalAlignment="Center">
-                                        <views:EquipmentCategoryIcon.Style>
```

Comment:
Style here is unnecessary, Generic.EquipmentCategoryIcon.xaml already collapses when Category is null